wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/rest/wahltage/WahltageController.java

```diff
@@ -0,0 +1,43 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.rest.wahltage;
+
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.services.wahltag.WahltageService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/businessActions/wahltage")
+@RequiredArgsConstructor
+@Slf4j
+public class WahltageController {
+
+    private final WahltageService wahltageService;
+    private final WahltageDTOMapper wahltageDTOMapper;
+
+    @Operation(description = "Laden der Liste der Wahltage, bei denen das Datum bis 3 Monate in die Vergangenheit reicht")
+    @ApiResponses(
+            value = {
+                    @ApiResponse(
+                            responseCode = "200", description = "OK",
+                            content = { @Content(mediaType = "application/json", array = @ArraySchema(schema = @Schema(implementation = WahltagDTO.class))) }
+                    )
+            }
+    )
+    @GetMapping
+    public ResponseEntity<List<WahltagDTO>> getWahltage() {
+        return new ResponseEntity<>(
+                wahltageDTOMapper.fromListOfWahltagModelToListOfWahltagDTO(wahltageService.getWahltage()),
+                HttpStatus.OK);
```

Comment:
Eventuell muss noch spotless drüber laufen

```suggestion
    public List<WahltagDTO> getWahltage() {
        return wahltageDTOMapper.fromListOfWahltagModelToListOfWahltagDTO(wahltageService.getWahltage());
```

Reply:
Bei mir in IntelliJ wird ordentlich dargestellt

Reply:
ResponseEntity braucht man nur wenn man programmatisch an der Response etwas ändern will. Dies kann der Fall sein wenn je nach Response ein anderer StatusCode oder Header zu setzen ist. Ist die Antwort aber Fix kann man die ResponseEntity weglassen und den Status über `org.springframework.web.bind.annotation.ResponseStatus` setzen, wobei `OK` der Default ist und nicht annotiert werden muss.